openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -581,6 +589,115 @@ def create_heketi_volume_with_name_and_wait(
 
         return h_volume_info
 
+    def create_heketi_volumes_in_batch(
```

Comment:
Need to call [1] function in case we create block hosting volume i.e if block=True
[1] https://github.com/gluster/glusterfs-containers-tests/pull/43/files#diff-5a2ebadc58342f9f6c9f0546e750812100bc44a87ce0227d8491dbd3ce2fd148R927

Reply:
done